src/components/molecules/Settings/Project/PublishSection/index.tsx

```diff
@@ -71,15 +70,19 @@ const PublishSection: React.FC<Props> = ({
   return (
     <>
       <Wrapper>
-        <Section title={intl.formatMessage({ defaultMessage: "Site settings" })}>
+        <Section title={intl.formatMessage({ defaultMessage: "Domain Settings" })}>
           <StyledText size="m" color={theme.main.strongText}>
-            {intl.formatMessage({ defaultMessage: "Site name" })}
+            {intl.formatMessage({ defaultMessage: "Default Domain" })}
```

Comment:
```suggestion
            {intl.formatMessage({ defaultMessage: "Site name" })}
```

"Domain" is not suitable. Custom domain setting is available in only SaaS. In OSS, published page URL will be `https://test.reearth.dev/p/HOGEHOGE` so HOGEHOGE is no longer "domain". "Site name" is good.